oak_containers_kernel/configs/6.1.33/minimal.config

```diff
@@ -12,7 +12,6 @@ CONFIG_LD_IS_BFD=y
 CONFIG_LD_VERSION=24100
 CONFIG_LLD_VERSION=0
 CONFIG_CC_CAN_LINK=y
-CONFIG_CC_CAN_LINK_STATIC=y
```

Comment:
@andrisaar @conradgrobler not sure about this diff, it came up after I built things locally, is it OK?

Reply:
I think it depends on whether glibc-static is available. I am not sure why this change would have caused it, but I don't think will break anything.